### ISSUE TITLE
How to enter multiple search patterns?

### ISSUE THREAD

**Issue by pstein** [deadlydog/PathLengthChecker#74]
Assume I want to list only files which match the 3 different patterns then entering the following does not work:

*.pdf *.log *.jpg

Even when I concatenate them by a comma link in

**Comment by deadlydog** [comment]
Yeah, right now the API being used behind the scenes to search for files only supports a single search pattern; you can't provide multiple. You'd need to do the search 3 separate times, using `*.pdf` the first time, `*.log` the second time, and `*.jpg` the third time.

For the immediate time, I could update the tooltip and docs to mention that only a single search pattern is supported.

For the future, we may look at doing the filtering in the app to allow multiple search patterns, as well as being able to exclude certain patterns as well as suggested in #72. That would take a bit more dev work though. PRs are welcomed ;)
